README.md

```diff
@@ -221,18 +221,19 @@ curl -sS https://starship.rs/install.sh | sh
 
 Alternatively, install Starship using any of the following package managers:
 
-| Distribution       | Repository              | Instructions                                                  |
-| ------------------ | ----------------------- | ------------------------------------------------------------- |
-| **_Any_**          | **[crates.io]**         | `cargo install starship --locked`                             |
-| _Any_              | [conda-forge]           | `conda install -c conda-forge starship`                       |
-| _Any_              | [Linuxbrew]             | `brew install starship`                                       |
-| Alpine Linux 3.13+ | [Alpine Linux Packages] | `apk add starship`                                            |
-| Arch Linux         | [Arch Linux Extra]      | `pacman -S starship`                                          |
-| CentOS 7+          | [Copr]                  | `dnf copr enable atim/starship` <br /> `dnf install starship` |
-| Gentoo             | [Gentoo Packages]       | `emerge app-shells/starship`                                  |
-| Manjaro            |                         | `pacman -S starship`                                          |
-| NixOS              | [nixpkgs]               | `nix-env -iA nixpkgs.starship`                                |
-| Void Linux         | [Void Linux Packages]   | `xbps-install -S starship`                                    |
+| Distribution        | Repository              | Instructions                                                  |
+| ------------------- | ----------------------- | ------------------------------------------------------------- |
+| **_Any_**           | **[crates.io]**         | `cargo install starship --locked`                             |
+| _Any_               | [conda-forge]           | `conda install -c conda-forge starship`                       |
+| _Any_               | [Linuxbrew]             | `brew install starship`                                       |
+| Alpine Linux 3.13+  | [Alpine Linux Packages] | `apk add starship`                                            |
+| Arch Linux          | [Arch Linux Extra]      | `pacman -S starship`                                          |
+| CentOS 7+           | [Copr]                  | `dnf copr enable atim/starship` <br /> `dnf install starship` |
+| Gentoo              | [Gentoo Packages]       | `emerge app-shells/starship`                                  |
+| Manjaro             |                         | `pacman -S starship`                                          |
+| NixOS               | [nixpkgs]               | `nix-env -iA nixpkgs.starship`                                |
+| openSUSE Tumbleweed | [OSS]                   | `zypper in starship`                                          |
```

Comment:
It also seems to be in [leap](https://repology.org/project/starship/versions), please just write "openSUSE" without the Tumbleweed suffix.

Reply:
Sure, can do that though I cannot test it on my machine. I deliberately left it out because [here on software.opensuse.org](https://software.opensuse.org/package/starship) it was marked as experimental for Leap.

Reply:
I just confirmed that it works and installs fine in the leap docker image.

Reply:
Super, thanks for double checking for that. :)